src/eatery/common_eatery.py

```diff
@@ -115,6 +117,25 @@ def parse_payment_methods(methods):
   payment_methods.swipes = any(pay['descrshort'] == PAY_METHODS['swipes'] for pay in methods)
   return payment_methods
 
+def parse_payment_methods_enum(methods):
+  """Returns a PaymentMethoddsEnumType according to which payment methods are available at an
+  eatery.
+
+  Args:
+      methods (json): a valid json segment for payments from Cornell Dining
+  """
+  payment_methods = []
+  if (any(pay['descrshort'] == PAY_METHODS['brbs'] for pay in methods)):
+      payment_methods.append(PaymentMethodsEnum.BRB)
+  if (any(pay['descrshort'] == PAY_METHODS['credit'] for pay in methods)):
+      payment_methods.append(PaymentMethodsEnum.CASH)
+      payment_methods.append(PaymentMethodsEnum.CREDIT)
+  if (any(pay['descrshort'] == PAY_METHODS['c-card'] for pay in methods)):
+      payment_methods.append(PaymentMethodsEnum.MOBILE)
+  if (any(pay['descrshort'] == PAY_METHODS['swipes'] for pay in methods)):
+      payment_methods.append(PaymentMethodsEnum.SWIPES)
+  return payment_methods
```

Comment:
probably cleaner to:

```
for method in methods:
  description = method['descrshort']
  if description == PAY_METHODS['brbs']:
    ...
```

Reply:
That makes a lot more sense, although we already have two queries (campusEatery and eateries) that give the exact same information. Should the new field be part of one of them or both of them?